desktop/main/Networks.ts

```diff
@@ -11,7 +12,8 @@ import { toPublicService } from './utils';
 //
 
 export const generateGenesisID = (genesisTime: string, extraData: string) => {
-  return `${toHexString(hash(genesisTime + extraData)).substring(0, 40)}`;
+  const time = parseISODate(genesisTime).getTime() / 1000;
```

Comment:
parseISODate returns milliseconds?

Reply:
Yeah

Agree about fetching from grpc, but we're indexing networks by genesis id (auto-connect to them, store logs using it and etc) — so we need it earlier (and even if Node does not work).

But probably Smapp should show some alerts in case the calculated genesis id and the one returned from API are different.